scripts/ci/verify_conda_install.sh

```diff
@@ -5,7 +5,7 @@ set -e #exit on error
 
 export VERSION="$(echo "$(ls dist/*.whl)" | cut -d- -f2)"
 
-conda install --no-deps "$CONDA_PREFIX/conda-bld/noarch/bokeh-$VERSION-py_0.tar.bz2"
+conda install --no-deps "$CONDA_PREFIX/conda-bld/noarch/bokeh-$VERSION-py_0.conda.tar.bz2"
```

Comment:
```suggestion
conda install --no-deps "$CONDA_PREFIX/conda-bld/noarch/bokeh-$VERSION-py_0.tar.bz2"
```

The `.conda` suffix is for the package format 2. Can be enabled with doing the following in `build_conda_package.sh`

``` bash
conda build conda/recipe --channel=defaults --no-test --no-anaconda-upload --no-verify --package-format 2
```

Reply:
I downgraded `conda-build` to `24*`, which doesn't enable v2 format by default. I started issue #14759 to get our build/release workflows updated.